### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by mebagel**

### CODE REVIEW

**Review by phil-pona — APPROVED** [review]

File: content/en/docs/01/labs/11-file-sd.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ In this first lab you are going to configure Prometheus to scrape the OpenShift-
 
 Before we get started, take the time to familiarize yourself with the config repository of your team - it should already be available as described in [Deploying the Baloise Monitoring Stack](https://confluence.baloisenet.com/atlassian/display/BALMATE/01+-+Deploying+the+Baloise+Monitoring+Stack).
 
-The working directory for this training is the folder in your team's config repository with the `-monitoring` suffix.
+The working directory for this training is the folder in your [team's config repository](http://{{% param replacePlaceholder.git %}}) with the `-monitoring` suffix. If necessary, create the directory `{{% param replacePlaceholder.git %}}-monitoring`.
```
Comment (madchr1st) [inline content/en/docs/01/labs/11-file-sd.md:15]: Still not working... don't see why not.

File: content/en/docs/01/labs/11-file-sd.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ In our example we added the host `myhost1.balgroupit.com` with an exporter runni
 
 As soon as the ConfigMap has been synchronized by ArgoCD, your defined targets should appear in Prometheus in the "Status -> Targets" submenu.
 
-Verify under <http://{{% param replacePlaceholder.prometheus %}}>.
+Verify under <http://{{% param replacePlaceholder.thanosquerier %}}>.
```
Comment (madchr1st) [inline content/en/docs/01/labs/11-file-sd.md:49]:
```suggestion
Verify in the [web UI](http://{{% param replacePlaceholder.thanosquerier %}}).
```

**Review by madchr1st — CHANGES_REQUESTED** [review]
Minor changes requested

File: content/en/docs/01/labs/11-file-sd.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ In our example we added the host `myhost1.balgroupit.com` with an exporter runni
 
 As soon as the ConfigMap has been synchronized by ArgoCD, your defined targets should appear in Prometheus in the "Status -> Targets" submenu.
 
-Verify under <http://{{% param replacePlaceholder.prometheus %}}>.
+Verify in the [web UI](http://{{% param replacePlaceholder.thanosquerier %}}).
```
Comment (madchr1st) [inline content/en/docs/01/labs/11-file-sd.md:49]:
```suggestion
Verify in the [web UI](http://{{% param replacePlaceholder.prometheus %}}).
```

**Review by madchr1st — CHANGES_REQUESTED** [review]
Last change request, rest looks good.

**Review by madchr1st — APPROVED** [review]
Nice, thx!

**Pull request merged**